app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Using a class-level dictionary `people` to store instances of `Person` can lead to issues if `create_person_list` is called multiple times, as it will accumulate all instances across calls. Consider using an instance-level attribute or returning a new list of `Person` instances instead.